src/awkward/operations/ak_to_categorical.py

```diff
@@ -55,16 +55,19 @@ def to_categorical(array, *, highlevel=True, behavior=None):
         ...     {"x": 1.1, "y": "one"}
         ... ])
         >>> records
-        <Array [{x: 1.1, y: 'one'}, ... y: 'one'}] type='5 * {"x": float64, "y": string}'>
+            <Array [{x: 1.1, y: 'one'}, ..., {x: 1.1, ...}] type='5 * {x: float64, y: s...'>
         >>> categorical_records = ak.zip({
         ...     "x": ak.to_categorical(records["x"]),
         ...     "y": ak.to_categorical(records["y"]),
         ... })
         >>> categorical_records
         <Array [{x: 1.1, y: 'one'}, ... y: 'one'}] type='5 * {"x": categorical[type=floa...'>
-        >>> ak.type(categorical_records)
-        5 * {"x": categorical[type=float64], "y": categorical[type=string]}
-        >>> ak.to_list(categorical_records) == ak.to_list(records)
+        >>> categorical_records.type.show()
+        5 * {
+            x: categorical[type=float64],
+            y: categorical[type=string]
+        }
+        >>> categorical_records.tolist() == records.tolist()
```

Comment:
If that's what we're doing, here's another one.

```suggestion
        >>> categorical_records.to_list() == records.to_list()
```